components/kyma-environment-broker/internal/cluster_config/gcp_provider.go

```diff
@@ -1,15 +1,13 @@
-package broker
+package cluster_config
```

Comment:
can we name that package `provider` and structs:
`GCPInput`
`AWSInput`
`AzureInput`
?